test/metrics_unittest.cc

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "../../src/metrics.h"
+
+#include <opencensus/stats/stats.h>
+#include <opencensus/stats/testing/test_utils.h>
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace google {
+
+TEST(SerializeToPrometheusTextTest, NoMetricExists) {
+  EXPECT_EQ("",
+            ::google::Metrics::SerializeMetricsToPrometheusTextFormat());
+}
+
+TEST(SerializeToPrometheusTextTest, SingleOpencensusView) {
+  const char measure_name[] = "test_measure";
+  const char view_name[] = "test_view";
+  const ::opencensus::stats::MeasureInt64 test_measure =
```

Comment:
Would it make sense to define the following in the `Metrics` class and use it here:
```c++
::opencensus::stats::MeasureInt64 Metrics::RegisterInt64(
    const std::string& name, const std::string& description,
    const std::string& unit) {
  const auto measure = ::opencensus::stats::MeasureInt64::Register(
    name, description, unit);
  const auto view_descriptor =
      ::opencensus::stats::ViewDescriptor()
          .set_name(name)
          .set_description(description)
          .set_measure(name)
          .set_aggregation(opencensus::stats::Aggregation::Count());
  view_descriptor.RegisterForExport();
  return measure;
}
```
? Then this test would become:
```c++
  const char* measure_name = "test_measure";
  const ::opencensus::stats::MeasureInt64 test_measure =
      Metrics::RegisterInt64(measure_name, "description_on_test_view", "1");
  ::opencensus::stats::testing::TestUtils::Flush();

  ::opencensus::stats::Record({{test_measure, 1}});
  ::opencensus::stats::testing::TestUtils::Flush();

  EXPECT_THAT(...);

  ::opencensus::stats::StatsExporter::RemoveView(measure_name);
```